doc/kb_sdk_complete_module_info.md

```diff
@@ -90,8 +90,10 @@ The fields that should be configured are
 - publications
 - screenshots
 
+##### Name, Icon, Tooltip, Description
```

Comment:
@dcchivian, @psdehal: See also https://github.com/kbase/project_guides/blob/master/Method_man_page.md.
Should we consider merging these docs?